project/build.properties

```diff
@@ -1 +1 @@
-sbt.version=0.13.13
+sbt.version=0.13.16
```

Comment:
Not if you want to cross-build. See http://www.scala-sbt.org/1.0/docs/Cross-Build-Plugins.html